cmd/cleanup.go

```diff
@@ -23,13 +23,10 @@ func getIDs(dockerOutput []byte) []string {
 	var result []string
 	outputArray := strings.Split(string(dockerOutput), "\n")
 
-	for _, entry := range outputArray {
-		// Each container in the Docker Compose stack will have the name of the directory
-		if strings.Contains(entry, otelConfigDirName) {
-			fields := strings.Fields(entry)
-			if len(fields) > 0 {
-				result = append(result, fields[0])
-			}
+	for _, entry := range outputArray[1:] {
```

Comment:
Why from `[1:]` instead of the whole thing now?

Reply:
Because line 0 contains the header line

Reply:
And there is no `--no-header` flag we could set

Reply:
sure, thought about that as well

Reply:
Resolved with f359289